docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+variance/error arrays that are propagated through these pipelines.
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Cube_build
+~~~~~~~~~~
+Cube_build takes MIRI or NIRSpec IFU calibrated 2-D images and produces 3-D
+spectral cubes.  In the output spectral cube, the SCI exension contains the
+surface brightness of cube spaxels in units of mJy/arcsecond^2, and the ERR
+extension contains the uncertainty on the SCI values.
+
+
+Stage 3 Coronographic Processing Pipeline - calwebb_coron3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The stage 3 coronagraphic pipeline is to be applied to associations of
+calibrated NIRCam coronagraphic and MIRI Lyot and 4QPM exposures, and is used 
+to produce PSF-subtracted, resampled, combined images of the source object.  The
+steps in this pipeline are stack_refs, align_refs, klip, outlier_detection, and
+resample.  The only one of these steps that either uses or modifies variance/error
+arrays that are propagated through the pipeline is outlier_detection, described below.
+For more details on this pipeline see `Calwebb_coron3
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_coron3.html>`_
+
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays. 
+
+
+Stage 3 Time-Series Observation (TSO) Processing Pipeline - calwebb_tso3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The stage 3 TSO pipeline is to be applied to associations of calibrated TSO
+exposures.  The steps in this pipeline are outlier_detection, tso_photometry,
+extract_1d, and white_light.  The only one of these steps that either uses or
+modifies variance/error arrays that are propagated through the pipeline is
+outlier_detection, described here:
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Stage 3 WFS&C Processing pipeline - calwebb_wfs-image3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Stage 3 processing of Wavefront Sensing and Control (WFS&C) images is only
+performed for dithered pairs of WFS&C exposures. The processing applied is not 
+truly a “pipeline”, but consists only of the single wfs_combine step. This step
+could modify variance/error arrays, but only if the optional 'do_refine' is set
+to True (which is *NOT* the default in pipeline use). In this case the "refined 
+algorithm" will be used, and the ERR array values will be altered on output.
+
+
+Other pipelines
```

Comment:
Don't need this section, because "calwebb_dark" quits before the ``dark_current`` step and hence it never gets to either ``ramp_fit`` or ``gain_scale``, which are the only 2 Stage 1 steps that do anything with ERR/VAR arrays.